src/packet_analysis/protocol/gre/GRE.cc

```diff
@@ -150,29 +152,39 @@ bool GREAnalyzer::AnalyzePacket(size_t len, const uint8_t* data, Packet* packet)
 				}
 			}
 
-		else if ( proto_typ == 0x8200 )
+		else if ( ((proto_typ & 0x8200) == 0x8200 && (proto_typ & 0x0F) == 0) ||
```

Comment:
> This is essentially what wireshark has this big list for? They call it "ARUBA WLAN"

Yep.

> Other approaches here would be to register 802.11 directly with GRE for all these ids in script land?

The problem with registering it with IEEE 802.11 is that you then have to duplicate some of the tunnel/header handling in that analyzer, when it really should be in the GRE analyzer. We're already handling other protocol types by number here as well.

Reply:
Never mind, I see what you meant. Yeah, it would be more correct to add the forwarding rules based on the types there. I'll have to see what setting `packet->proto` to one of the aruba types does downstream though.